src/main/java/mate/academy/spring/dao/impl/OrderDaoImpl.java

```diff
@@ -26,6 +26,8 @@ public OrderDaoImpl(SessionFactory factory) {
                             + "join fetch t.movieSession ms "
                             + "join fetch ms.cinemaHall "
                             + "join fetch ms.movie "
+                            + "join fetch o.user u "
+                            + "inner join fetch u.roles "
```

Comment:
Why inner?

Reply:
so that we retrieve  users with no roles